tests/kernel/tickless/tickless_concept/boards/nucleo_h723zg.conf

```diff
@@ -0,0 +1 @@
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=2048
```

Comment:
Fixing `CONFIG_SYS_CLOCK_TICKS_PER_SEC` value here may fix the test, but won't have any effect on behavior in applications.
Please see how this is managed for other STM32 boards, for instance nucleo_wb55rg or disco_l475_iot1.
(Also it would be interesting to know why default setting was not working).

Reply:
> Fixing `CONFIG_SYS_CLOCK_TICKS_PER_SEC` value here may fix the test, but won't have any effect on faulty behavior in applications (which the test is only revealing). Please see how this is managed for other STM32 boards, for instance nucleo_wb55rg or disco_l475_iot1. (Also it would be interesting to know why default setting was not working).

Reply:
I till take a look into it.